plugins/user/cronjobs

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env python3
+# -*- python -*-
+# -*- coding: utf-8 -*-
+#
+# Plugin to monitor the number of cronjobs running per user
+#
```

Comment:
Since the plugin uses the content of `/var/log/syslog`, you should mention this in the plugin description. This gives the users of this plugin a better idea, under which circumstances this plugin works for them.

Reply:
Done.